src/battle-log.ts

```diff
@@ -843,9 +843,15 @@ class BattleLog {
 				const videoId = /(?:\?v=|\/embed\/)([A-Za-z0-9_\-]+)/.exec(src)?.[1];
 				if (!videoId) return {tagName: 'img', attribs: ['alt', `invalid src for <youtube>`]};
 
+				const time = getAttrib('start') || /(?:\?|&)t=([0-9]+)/.exec(src)?.[1] || false;
```

Comment:
```suggestion
				const time = getAttrib('start') || /(?:\?|&)t=([0-9a-z]+)/.exec(src)?.[1] || false;
```

We should probably support hms notation.